### ISSUE TITLE
how can use STK681-050

### ISSUE THREAD

**Issue by dnshvra** [misan/dcservo#48]
hello
i have STK681-050 ic, Bidirectional DC Brush-Type Motor Driver (IO=5A).
this driver have 4 pin , INA, INB, INC, IND.
how can connect dcservo to this driver?
[STK681-050.pdf](https://github.com/misan/dcservo/files/2203587/STK681-050.pdf)
![image_001](https://user-images.githubusercontent.com/5108982/42848716-c5b2fb96-8a35-11e8-92eb-02874d730692.gif)

**Comment by dnshvra** [comment]
How to connect bts7960 module?

**Comment by misan** [comment]
There are several possible choices but you can use this one (and select the right output pins and this is borrowed from Arduino forums so PWM output pins may not match my code).

![image](https://user-images.githubusercontent.com/466120/42862777-3ed72634-8a61-11e8-94af-761c5cb366f1.png)

**Comment by langwadt** [comment]
you need to connect GND

**Comment by misan** [comment]
Absolutely. Arduino GND and other supplies GND is to be connected together
to keep magic smoke inside.

**Comment by PierreAuge** [comment]
https://github.com/PierreAuge/dcservo/blob/master/dcservo/DCServo-IBT2.ino

Here's a link to my working sketch using the IBT-2

**Comment by misan** [comment]
Hi @PierreAuge,

Good to know.  With that, you can drive a powerful motor. I would love to see it in action.
